org.lflang/src/org/lflang/validation/LFValidator.java

```diff
@@ -1394,31 +1418,42 @@ public void checkStateResetWithoutInitialValue(StateVar state) {
     //////////////////////////////////////////////////////////////
     //// Public methods.
 
-    /** 
-     * Return the error reporter for this validator. 
+    /**
+     * Return the error reporter for this validator.
      */
     public ValidatorErrorReporter getErrorReporter() {
         return this.errorReporter;
     }
-    
+
     /**
      * Implementation required by xtext to report validation errors.
      */
     @Override
     public ValidationMessageAcceptor getMessageAcceptor() {
         return messageAcceptor == null ? this : messageAcceptor;
     }
-    
+
     /**
      * Return a list of error messages for the target declaration.
      */
     public List<String> getTargetPropertyErrors() {
         return this.targetPropertyErrors;
     }
 
+    /**
+     * Generate an error message for an AST node.
+     * Overriding the parent method here to make the method public,
```

Comment:
```suggestion
```
This type of info doesn't really belong in Javadoc because it doesn't help a programmer decide whether (or what for) to use the method.